components/organisms/ProtocolMarket.tsx

```diff
@@ -43,3 +49,10 @@ export const ProtocolTokensMarket = () => {
         <TokenRowMarket address={"0x..."} name={"USD Coin"} volumen={"5.57"} borrowAPY={"3.26"} depositAPY={"2.19"} rewardAaveAPY={"0.96"} />
     </View>;
 };
+const styles = StyleSheet.create({
+    divider: {
+        height: 0.5,
```

Comment:
Can we change it to 1? In some cases using 0.5 it doesn't show anything on some devices

Reply:
reddy